packages/typespec-ts/src/index.ts

```diff
@@ -95,14 +95,17 @@ import { emitSamples } from "./modular/emitSamples.js";
 export * from "./lib.js";
 
 export async function $onEmit(context: EmitContext) {
+  console.time("onEmit");
```

Comment:
do we plan to remove this code? i would expect console.time/endtime is for debugging and should not happen in production or released code?

Reply:
Can we link this function call with logging level?  We could only display this info when we need the full trace.

Reply:
I plan to keep this for now, and will using tracing when I get to Dapeng's issue for error handling https://github.com/Azure/autorest.typescript/issues/3064